autogen/variables.tf

```diff
@@ -376,4 +376,11 @@ variable "enable_intranode_visibility" {
   description = "Vertical Pod Autoscaling automatically adjusts the resources of pods controlled by it"
   default     = false
 }
+
+variable "identity_namespace" {
+  type        = string
+  description = "Workload Identity namespace"
+  default     = ""
```

Comment:
Default should be `null`, right?

Reply:
Not sure, I've seen service_account var is string and "" by default as well.

Reply:
That was added before Terraform 0.12. Now that Terraform 0.12 has `null` we should use it.